src/main/java/com/sequenceiq/cloudbreak/domain/InstanceMetaData.java

```diff
@@ -36,11 +36,20 @@
     private Integer containerCount = 0;
     @ManyToOne
     private Stack stack;
+    private String instanceGroup;
```

Comment:
I think this should be a real db relation, because we cannot retrieve the instanceMetadata for an InstanceGroup if there are multiple InstanceGroups in an account with the same name (e.g.: master)